encoders/nlp/TransformerTorchEncoder/__init__.py

```diff
@@ -120,9 +139,27 @@ def amp_accelerate(self):
         else:
             return nullcontext()
 
-    def _compute_embedding_from_model_output(self, outputs: 'BaseModelOutput', input_tokens: Dict):
+    def _api_call(self, query):
```

Comment:
what is the return type? How from `json` is it so easy to convert to `Torch.Tensor`?

Reply:
Because it is already a list of a list of floats. Just the right format.